### ISSUE TITLE
Add one-click connect for Zap Desktop wallet via an lndconnect: link

### ISSUE THREAD

**Issue by mrfelton** [PierreRochard/node-launcher#81]
Zap Desktop now supports part of the `lndconnect:` protocol link format that can enable a smoother node connection process for users of Zap Desktop.

There are some docs here: https://github.com/LN-Zap/zap-iOS/blob/master/docs/lnd_connect_uri.md

Essentially, if you trigger the opening of a url that follows the `lndconnect` format, it will trigger ZapDesktop to open with all of the connection info already filled out. The user then just needs to confirm the details in ZapDesktop and submit the form.

eg:
```
lndconnect:?cert=/path/to/cert&macaroon=/path/to/macaroon&host=1.2.3.4:10009
```

In the future we will also enable the ability to pass the actual encoded cert and macaroon data directly in the link, but for the time being we just support paths to files.

We plan to update https://github.com/LN-Zap/zapconnect to support this new connection format, so you could then embed zapdesktop within your app and just use that to generate the properly encoded connection string.

**Comment by manecosta** [comment]
I noticed that the iOS app supports deep linking with those urls also which is very useful.

I provided @PierreRochard with some code that he already added to [https://github.com/PierreRochard/node-launcher/issues/54](https://github.com/PierreRochard/node-launcher/issues/54) to generate the lndconnect string and the qrcode based on the reference javascript implementation here [https://github.com/LN-Zap/lndconnect/blob/master/lnd_connect_uri.md](https://github.com/LN-Zap/lndconnect/blob/master/lnd_connect_uri.md)

I haven't been able to make it work on iOS though but once I can I'll submit a pull request trying to add this functionality.

**Comment by mrfelton** [comment]
@manecosta the documentation in https://github.com/LN-Zap/lndconnect/blob/master/lnd_connect_uri.md refers to a slightly updated version of lndconnect which is not yet fully supported in Zap Desktop.

For ZapDesktop, the format should be as per my previous comment for the time being.

**Comment by manecosta** [comment]
Ok, that's actualy simpler as there's no chance to mess it up when file parsing and base64 encoding.

**Comment by manecosta** [comment]
This issue is already addressed by #98

**Comment by PierreRochard** [comment]
Thank you @manecosta! Really grateful for your help
